### DISCUSSION TITLE
Payload Encryption

### DISCUSSION THREAD

**Discussion by tsndp** [livewire/livewire#1663] [1. Help]
Hi,

How can we implement payload encryption, the communication on both ways is encrypted (request and response) with aes rsa.

note: please leave ssl for this case.

**Comment by tsndp** [comment]
bump

**Comment by joshhanley** [comment]
@tsndp can you provide some more information? What is your use case? What are you trying to achieve and why?

All your data should be running over https, is that not sufficient for your needs?

Are you talking about encrypting data that is sent back and forth to the browser? If so have you done this in other apps (ie in VueJS), and how was that implemented?

If you provide some more information, you are more likely to get a response. I suspect most people just don't understand what your asking.

Hope this helps!
